firmware/Core/Src/telecommands/telecommand_executor.c

```diff
@@ -43,8 +54,8 @@ const char* telecommand_channel_enum_to_str(TCMD_TelecommandChannel_enum_t chann
 /// @return 0 on success, 1 if the agenda is full.
 /// @note Performs a deep copy of the `parsed_tcmd` arg into the agenda.
 uint8_t TCMD_add_tcmd_to_agenda(const TCMD_parsed_tcmd_to_execute_t *parsed_tcmd) {
-    // FIXME: Issue #53 - Check `parsed_tcmd.timestamp_sent` against the yet-to-be-created `TCMD_timestamp_sent_record[]`, and skip this command if it is (to prevent replays).
-    // ^ Use a config variable to override this check, if desired.
+    // FIXME: If anyone wants please create unit tests for this. I am not sure if I should come back and add one but 
+    //we will see.
```

Comment:
Please rephrase in the format of:

> TODO: Create unit test for <explain what the unit test is for>, if desired

I'd suggest that this might not be a place that a unit test makes a lot of sense